crawlers/mooncrawl/mooncrawl/stats_worker/dashboard.py

```diff
@@ -652,25 +658,26 @@ def stats_generate_handler(args: argparse.Namespace):
 
         address_dashboard_id_subscription_id_tree: Dict[str, Any] = {}
 
-        for user_id, collection_id in user_collection_by_id.items():
+        for user_id, journal_id in user_collection_by_id.items():
             # request all subscriptions for user
 
-            user_subscriptions: EntityCollectionResponse = ec.search_entities(
+            user_subscriptions = bc.search(
                 token=MOONSTREAM_ADMIN_ACCESS_TOKEN,
-                collection_id=collection_id,
-                required_field=[
-                    "subscription_type_id:{}".format(
-                        subscription_id_by_blockchain[args.blockchain]
-                    )
-                ],
+                journal_id=journal_id,
+                query=f"tag:subscription_type_id:{subscription_id_by_blockchain[args.blockchain]}",
+                representation="entity",
             )
 
+            user_subscriptions_results = cast(
+                List[BugoutSearchResultAsEntity], user_subscriptions.results
+            )
             logger.info(
-                f"Amount of user subscriptions: {len(user_subscriptions.entities)}"
+                f"Amount of user subscriptions: {len(user_subscriptions_results)}"
             )
 
-            for subscription in user_subscriptions.entities:
-                subscription_id = str(subscription.entity_id)
+            for subscription in user_subscriptions_results:
+                entity_url_list = subscription.entity_url.split("/")
```

Comment:
Can be just [-1] instead of [len(entity_url_list) - 1]

Reply:
Its same